portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,11 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+async function fetchHelloWorld() {
+	document.getElementById("inner-content").style.display = "block";
```

Comment:
nit - this seems unrelated to this method (at least to the casual reader). Also, since this method is invoked on page load, would it make sense to just style this element with css?

Reply:
thanks! will do!